package.json

```diff
@@ -5,7 +5,7 @@
   "main": "src/index.html",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "browser-sync start --server --directory",
+    "start": "browser-sync start --server --directory --files='src/*.html'",
```

Comment:
Good idea! Maybe we should watch some _other_ files, too? /cc @TIY-Durham/2015-fall-fee